src/SAWScript/MGU.hs

```diff
@@ -124,7 +124,7 @@ listSubst = Subst . M.fromList
 --
 
 data FailMGU = FailMGU
-                    String      -- initial error message
+                    [String]    -- initial error message
```

Comment:
The comment mentions "error message" (singular), which would suggest a `String` rather than a `[String]`. Is there significance to the use of a list of `String`s here? If so, it would be worth updating the comment to explain what the list is used for.

Reply:
It's still singular but it may be more than one line. I'll clarify